utils/dataset_manifest/utils.py

```diff
@@ -21,4 +24,112 @@ def rotate_image(image, angle):
 def md5_hash(frame):
     if isinstance(frame, VideoFrame):
         frame = frame.to_image()
-    return hashlib.md5(frame.tobytes()).hexdigest() # nosec
\ No newline at end of file
+    return hashlib.md5(frame.tobytes()).hexdigest() # nosec
+
+def _define_data_type(media):
+    media_type, _ = mimetypes.guess_type(media)
+    if media_type:
+        return media_type.split('/')[0]
+
+def is_video(media_file):
+    return _define_data_type(media_file) == 'video'
+
+def is_image(media_file):
```

Comment:
We already have similar functions here: https://github.com/openvinotoolkit/cvat/blob/bd49d7ae99c66ebfec500e4a98af053cdc55e618/cvat/apps/engine/media_extractors.py#L607

Reply:
@nmanovic , This was my implementation (because we discussed that in the future this should be moved to Datumaro and the current implementation should not depend on CVAT), Boris just moved it from `create.py`

Reply:
@Marishka17 , does it handle svg images properly?

Reply:
Apparently not

Reply:
OK. Let's wait till we have a bug about that. :( 12 days and nothing was changed.

Reply:
OK. Let's wait till we have a bug about that. :( 12 days and nothing was changed.